Applications/Spire/Include/Spire/Ui/TableModel.hpp

```diff
@@ -127,6 +149,53 @@ namespace Details {
   const T& TableModel::get(int row, int column) const {
     return std::any_cast<const T&>(at(row, column));
   }
+
+  template<typename F>
+  void TableModel::visit(const TableModel::Operation& operation, F&& f) const {
+    if(operation.type() == typeid(Transaction)) {
```

Comment:
Use the declaration form of the if statement to avoid performing two tests:

```if(auto transaction = boost::get<Transaction>(&operation)) {```

https://en.cppreference.com/w/cpp/language/if